lib/parse/index.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * parse/index.js - JOSE Parser Entry Point
+ *
+ * Copyright (c) 2015 Cisco Systems, Inc. See LICENSE file.
+ */
+"use strict";
+
+var compact = require("./compact"),
+    json = require("./json");
+
+var parse = module.exports = function(input) {
+  if ("string" === typeof input) {
```

Comment:
Ok, in that case, `if (Buffer.isBuffer(input)) { input = input.toString(); }` should be fine. I couldn't remember if `Buffer` was used explicitly if they were `arraybuffers`. Any explicit use of `Buffer` will cause bundlers to automatically include the shim.